R/save_arcadia_plot.R

```diff
@@ -1,57 +1,86 @@
 #' Save Arcadia Plot
 #'
-#' @param plot ggplot object to be saved
+#' @param plot ggplot object to be saved as both a PDF and PNG
 #' @param filepath The path where the plot will be saved
```

Comment:
Should this be amended to say "without file extension"?

Reply:
addressed